kernel/packages/unity-interface/setFilteredConsoleError.ts

```diff
@@ -0,0 +1,16 @@
+let isOverridden = false
+
+const ignoredErrors = ['Asset Bundle download is complete, but no data have been received\n']
+
+export function setFilteredConsoleError() {
+  if (isOverridden) return
+
+  isOverridden = true
+  const defaultConsoleError = window.console.error
+
+  window.console.error = function () {
+    if (!ignoredErrors.includes(arguments[0])) {
```

Comment:
does this work if we write "console.error(null) or console.error(new Error())"?

Reply:
yeap, just tested that, it works as expected